### DISCUSSION TITLE
I am getting style ssr glitch in nextjs

### DISCUSSION THREAD

**Discussion by kingRayhan** [mantinedev/mantine#3255] [Feedback]
https://user-images.githubusercontent.com/7611746/209910589-ab67f6be-74c3-4ba5-8887-9b97b8a8e81c.mp4

I am working on a project, everything was going fantastic but all of a sudden I got this styling glitch. For recreating the issue I create another small mantine-next project but this project also catches the same style glitching problem.

Here is the reproduction project: https://github.com/kingRayhan/mantine-ecommerce-dashboard

**Comment by rtivital** [comment]
Follow next.js getting started guide, if you have issues use one of the templates provided in documentation as reference

**Comment by limichange** [comment]
I had the same problem, just upgrade the NextJS and Mantine versions and everything will be fine.
